obi_one/scientific/circuit/neuron_sets.py

```diff
@@ -5,9 +5,67 @@
 
 import bluepysnap as snap
 import numpy as np
+import pandas
 from pydantic import Field, model_validator
 
+from obi_one.core.base import OBIBaseModel
 from obi_one.core.block import Block
+from obi_one.scientific.circuit.circuit import Circuit
+
+
+class NeuronPropertyFilter(OBIBaseModel, abc.ABC):
+    filter_dict: dict[str, list[str]] = Field(
```

Comment:
We should not restrict to list of strings. E.g., in some older circuit releases, layers are still encoded as integers, not strings.
```suggestion
    filter_dict: dict[str, list] = Field(
```